Both usage and ouput documents updated and populated. Comments within… (#1)
wal-yan/picard-profiler: docs/usage.md

Author: Am I correct in assuming there should be a section on Samtools_faidx in here that I missed?

Reviewer: Yes, as this is an optional step, we need to highlight when this is useful for a user to invoke.

You will need to create a samplesheet with information about the samples you would like to analyse before running the pipeline. Use this parameter to specify its location. It has to be a comma-separated file with 3 columns, and a header row as shown in the examples below.

```bash
--samplesheet '[path to samplesheet file]'
```

### Full samplesheet

The pipeline requires the use of the compressed alignment files bam and bai, output from the nfcore/methylseq pipeline. The samplesheet requires three columns, which must follow the order given below.

```console
sample,bai,bam
CONTROL_REP1,AEG588A1_S1_L002_R1_001.markdup.sorted.bam.bai,AEG588A1_S1_L002_R2_001.markdup.sorted.bam
CONTROL_REP2,AEG588A2_S2_L002_R1_001.markdup.sorted.bam.bai,AEG588A2_S2_L002_R2_001.markdup.sorted.bam
CONTROL_REP3,AEG588A3_S3_L002_R1_001.markdup.sorted.bam.bai,AEG588A3_S3_L002_R2_001.markdup.sorted.bam

> # didnt change the file names, literally just changed the sample type to bam/bai. Used the same samples as the example that was already given.

Reviewer: This sounds good 👍 

```suggestion

```

```

| Column    | Description                                                                                                                                                                            |
| --------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| `sample`  | Custom sample name. This entry will be identical for multiple sequencing libraries/runs from the same sample. Spaces in sample names are automatically converted to underscores (`_`) `## is converting spaces to _ true in picard case? ##`. |
| `bai` | Full path to bai file for sample. File must be those that are "mark duplicate alignments" in sorted files from previous methylseq runs.                                                              |
| `bam` | Full path to bam file for sample. File must be those that are "mark duplicate alignments" in sorted files from previous methylseq runs.                                                             |

> An [example samplesheet](../assets/samplesheet.csv) has been provided with the pipeline. `# example samplesheet still needs to be uploaded`

Reviewer: ```suggestion
An [example samplesheet](../assets/test_samplesheet.csv) has been provided with the pipeline.
```

nextflow pull wal-yan/picard-profiler
```

> ### Reproducibility - how much of this should be edited? reading through this it sounds fine?

Reviewer: This is fine, yes 👍 

```suggestion
### Reproducibility
```

> 💡 If you wish to share such profile (such as upload as supplementary material for academic publications), make sure to NOT include cluster specific paths to files, nor institutional specific profiles.

> ## Core Nextflow arguments - assuming these all stay the same?

Reviewer: Yes, those come bundled with the nf-core template.

```suggestion
## Core Nextflow arguments 
```

Specify the path to a specific config file (this is a core Nextflow command). See the [nf-core website documentation](https://nf-co.re/usage/configuration) for more information.

> ## Custom configuration - unsure how much of this is still relevant for picard profiler? Does all of this still apply?

Reviewer: It does apply, but I don't think we need to alter the default setup in this case.

```suggestion
## Custom configuration
```
